src/amplitude.js

```diff
@@ -468,8 +599,16 @@ Amplitude.prototype.setDeviceId = function(deviceId) {
   }
 };
 
-Amplitude.prototype.setUserProperties = function(userProperties) {
-  if (!this._apiKeySet('setUserProperties()')) {
+/**
+ * Sets user properties for the current user.
+ * @public
+ * @param {object} - object with string keys and values for the user properties to set.
+ * @param {boolean} - DEPRECATED opt_replace: in earlier versions of the JS SDK the user properties object was kept in
```

Comment:
A lot of people have asked about where the `replace` flag went, so I added a little explanation